.travis.yml

```diff
@@ -3,37 +3,65 @@
 
 language: python
 
-# Run jobs on container-based infrastructure, can be overridden per job
-sudo: false
-dist: trusty
 
-python:
-    - 2.7
-    - 3.5
-    - 3.6
+matrix:
+  include:
+  # Test osx
+  - os: osx
+    language: generic
+    env: PYTHON_VER=2.7
```

Comment:
This will get into a lock state due to the lack of `ByteStream.read1` in py27. Remove py27 from mac tests